frontend/src/components/CreateCollectionModal/index.tsx

```diff
@@ -13,31 +14,60 @@ const AsyncContent = loadable(
     )
 );
 
+const AsyncCTA = loadable(
```

Comment:
Code split and lazy load CTA only when the condition is met, so we don't send down unnecessary bytes to users